meta-oe/recipes-printing/cups/cups-filters.inc

```diff
@@ -0,0 +1,100 @@
+
+SECTION = "console/utils"
+
+DEPENDS = "cups glib-2.0 glib-2.0-native dbus dbus-glib lcms ghostscript poppler qpdf libpng"
+DEPENDS_class-native = "poppler-native glib-2.0-native dbus-native pkgconfig-native gettext-native libpng-native"
+
+SRC_URI = "http://openprinting.org/download/cups-filters/cups-filters-${PV}.tar.gz"
+
+inherit autotools-brokensep gettext pkgconfig
+
+EXTRA_OECONF_append = " --enable-ghostscript --disable-ldap \
+                       --with-pdftops=hybrid --enable-imagefilters \
+                       --enable-ghostscript --with-gs-path=${bindir}/gs \
+                       --with-pdftops-path=${bindir}/gs \
+                       --with-fontdir=${datadir}/fonts --with-rcdir=no \
+                       --with-cups-rundir=${localstatedir}/run/cups \
+                       --localstatedir=${localstatedir}/var \
+                       --with-rcdir=no \
+                       --without-php"
+
+EXTRA_OECONF_append_class-native = " --with-pdftops=pdftops \
+                                    --disable-avahi --disable-ghostscript \
+                                    --disable-ldap \
+                                    --with-png --without-jpeg --without-tiff"
+
+EXTRA_OEMAKE = "-j1"
+
+BBCLASSEXTEND = "native"
+
+PACKAGECONFIG[jpeg] = "--with-jpeg,--without-jpeg,jpeg"
+PACKAGECONFIG[png] = "--with-png,--without-png,libpng"
+PACKAGECONFIG[tiff] = "--with-tiff,--without-tiff,tiff"
+
+PACKAGECONFIG ??= "${@bb.utils.contains('DISTRO_FEATURES', 'zeroconf', 'avahi', '', d)}"
+
+PACKAGECONFIG[avahi] = "--enable-avahi,--disable-avahi,avahi"
+PACKAGECONFIG[dbus] = "--enable-dbus,--disable-dbus,dbus"
+
+DIRFILES = "1"
+
+PACKAGES =+ "\
+	${PN}-gst \
+        ${PN}-data \
+        "
+
+FILES_${PN}-gst = "\
+	${libdir}/cups/filter/gstopxl \
+	${libdir}/cups/filter/gstoraster \
+	"
+
+FILES_${PN}-data = "\
+	/usr/share/cups/data \
+	"
+
+FILES_${PN}-dbg += "\
+	${libdir}/cups/filter/.debug \
+	${libdir}/cups/backend/.debug \
+	"
+
+FILES_${PN} += "\
+        ${libdir}/cups/filter \
+        ${libdir}/cups/backend \
+        ${libdir}/cups/driver \
+        /usr/share/cups/charsets \
+        /usr/share/cups/drv \
+        /usr/share/cups/mime \
+        /usr/share/cups/ppdc \
+        /usr/share/ppd/cupsfilters \
+        /usr/share/cups/braille \
+        /usr/share/cups/banners \
+        /usr/share/cups/braille/index.sh \
+        /usr/share/cups/braille/cups-braille.sh \
+        /usr/share/cups/braille/indexv3.sh \
+        /usr/share/cups/braille/indexv4.sh \
+        /usr/share/cups/banners/topsecret \
+        /usr/share/cups/banners/secret \
+        /usr/share/cups/banners/confidential \
+        /usr/share/cups/banners/unclassified \
+        /usr/share/cups/banners/form \
+        /usr/share/cups/banners/classified \
+        /usr/share/cups/banners/standard \
+"
+
+do_install_append() {
+	# remove banners, braille dirs
+	rm -rf ${D}/usr/share/cups/{banners,braille}
```

Comment:
Use `${datadir}` instead of hardcoding /usr/share

Reply:
Done.